cmd/clusterctl/client/cluster/cert_manager.go

```diff
@@ -269,27 +288,16 @@ func (cm *certManagerClient) EnsureLatestVersion(ctx context.Context) error {
 	}
 
 	// Install cert-manager.
-	return cm.install(ctx)
+	return cm.install(ctx, config.Version(), installObjs)
 }
 
-func (cm *certManagerClient) migrateCRDs(ctx context.Context) error {
-	config, err := cm.configClient.CertManager().Get()
-	if err != nil {
-		return err
-	}
-
-	// Gets the new cert-manager components from the repository.
-	objs, err := cm.getManifestObjs(ctx, config)
-	if err != nil {
-		return err
-	}
-
+func (cm *certManagerClient) migrateCRDs(ctx context.Context, installObj []unstructured.Unstructured) error {
 	c, err := cm.proxy.NewClient(ctx)
 	if err != nil {
 		return err
 	}
 
-	return NewCRDMigrator(c).Run(ctx, objs)
+	return NewCRDMigrator(c).Run(ctx, installObj)
```

Comment:
q: why are passing installed objects here?
If I remember well CRD migrator expect to get in input the objects that are going to be installed to determine what is going to change

Reply:
installObj are the objects to-be installed, not the one which exist :-)

It's the same as before, the output of: `cm.getManifestObjs(ctx, config)`